stable/dask/templates/_helpers.tpl

```diff
@@ -0,0 +1,31 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Please update with latest `_helpers.tpl` you get from `helm create` using Helm 2.8+.